docs/polaris/data-science-workflows/frameworks/rapids.md

```diff
@@ -0,0 +1,272 @@
+# Rapids on Polaris
```

Comment:
I think NVIDIA stylizes it like "RAPIDS"

Reply:
done